### ISSUE TITLE
Feature Request:  Fade out settings.

### ISSUE THREAD

**Issue by pixelrogue1** [FuzzyIdeas/IsThereNet#6]
Would love the ability to have the indicators last longer...adjusted by the user...including option to keep status indicator 100% of the time, useful for example when troubleshooting networks.

**Comment by alin23** [comment]
Thanks for the request! As you can see, the app is not configurable and I'd like to keep it that way as I don't have time to work on it. 

I'll say no to adding any kind of settings for now, although yours is a good idea. The issue will remain here, closed, for when I'll have time to develop this app further.

**Comment by pixelrogue1** [comment]
Thank you for the quick reply. It seemed to work once…that was it. Nothing to force with our write all, and relaunching wouldn’t do anything.There are other tools for this need that I will stick with for the present. Thank you again for the response and decision.On Mar 27, 2024, at 11:37 AM, Alin Panaitiu ***@***.***> wrote:﻿
Closed #6 as not planned.

**Comment by alin23** [comment]
This can now be adjusted in [v1.5](https://github.com/FuzzyIdeas/IsThereNet/releases/tag/v1.5)

```json
  "fadeSeconds" : {
    "connected" : 5,
    "disconnected" : 0,
    "slow" : 10
  }
```
